ruddock/templates/info.html

```diff
@@ -0,0 +1,31 @@
+{% extends "layout.html" %}
+{% block body %}
+<h2>Useful House Info</h2>
+<h3>Door codes</h3>
+<table>
+<tr><td>Computer lab</td><td>34-5-2-1</td></tr>
+<tr><td>PGR</td><td>1-34-2</td></tr>
+<tr><td>O'Doms</td><td>5-3-2-4</td></tr>
+<tr><td>Kitchens</td><td>24-3-1-5</td></tr>
+</table>
+<h3>Printers</h3>
+<p>If you need help setting up a printer, don't hesitate to ask any IMSS rep.</p>
+<table>
+<tr><th>Name</th><th>Color</th><th>Location</th><th>Network address</th><th>Model</th></tr>
+<tr><td>Desert</td><td>b&w</td><td>O'Doms</td><td>desert.caltech.edu</td><td>Xerox Phaser 4510</td></tr>
+<tr><td>Tundra</td><td>b&w</td><td>Computer Lab</td><td>tundra.caltech.edu</td><td>Xerox Phaser 4600</td></tr>
+<tr><td>Prairie</td><td>color</td><td>Computer Lab</td><td>prairie.caltech.edu</td><td>Xerox Phaser 6600</td></tr>
+</table>
+<h3>Mailing Lists</h3>
+<p>All lists are @ruddock.caltech.edu.</p>
+<table>
+<tr><th>Name</th><th>Notes</th></tr>
+<tr><td>spam@</td><td>General mailing list. RA's are on this list.</td></tr>
```

Comment:
There's disagreement on the subject: https://english.stackexchange.com/questions/503/what-is-the-correct-way-to-pluralize-an-acronym